src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -1676,33 +1533,6 @@ public GHPullRequest getPullRequest(int i) throws IOException {
         return root().createRequest().withUrlPath(getApiTailUrl("pulls/" + i)).fetch(GHPullRequest.class).wrapUp(this);
     }
 
-    /**
-     * Retrieves all the pull requests of a particular state.
-     *
-     * @param state
-     *            the state
-     * @return the pull requests
-     * @throws IOException
-     *             the io exception
-     * @see #listPullRequests(GHIssueState) #listPullRequests(GHIssueState)
-     */
-    public List<GHPullRequest> getPullRequests(GHIssueState state) throws IOException {
-        return queryPullRequests().state(state).list().toList();
-    }
-
-    /**
-     * Retrieves all the pull requests of a particular state.
-     *
-     * @param state
-     *            the state
-     * @return the paged iterable
-     * @deprecated Use {@link #queryPullRequests()}
-     */
-    @Deprecated
-    public PagedIterable<GHPullRequest> listPullRequests(GHIssueState state) {
-        return queryPullRequests().state(state).list();
-    }
-
```

Comment:
@ihrigb 
Noted that this was not marked as Deprecated.  

Did a search on usages:
https://github.com/search?q=org%3Ajenkinsci+getPullRequests+path%3A%2F%5Esrc%5C%2Fmain%5C%2Fjava%5C%2F%2F+org.kohsuke.github&type=code

It looks like there's at least one usage in the wild, and since it is in Jenkins it will cause breaks for some time to come. 
https://github.com/jenkinsci/ghprb-plugin/blob/master/src/main/java/org/jenkinsci/plugins/ghprb/GhprbRepository.java#L145

There's a similar one for listPullRequests().  

Might consider bringing these back at least as bridge methods to not break Jenkins plugins.  On the other hand this is a 2.0, accepting some breakages might be acceptable.